src/charm.py

```diff
@@ -643,50 +691,44 @@ def _push_config_file_to_workload(self, content: str):
         self._container.push(path=VAULT_CONFIG_FILE_PATH, source=content)
         logger.info("Pushed %s config file", VAULT_CONFIG_FILE_PATH)
 
-    def _set_certificates_secret_in_peer_relation(
+    def _set_ca_certificate_secret_in_peer_relation(
         self,
         private_key: str,
         certificate: str,
-        ca_certificate: str,
     ) -> None:
-        """Set the vault certificate secret in the peer relation.
+        """Set the vault CA certificate secret in the peer relation.
 
         Args:
             private_key: Private key
             certificate: certificate
-            ca_certificate: CA certificate
         """
         if not self._is_peer_relation_created():
             raise RuntimeError("Peer relation not created")
         juju_secret_content = {
             "privatekey": private_key,
             "certificate": certificate,
-            "cacertificate": ca_certificate,
         }
-        juju_secret = self.app.add_secret(juju_secret_content, label="vault-certificate")
+        juju_secret = self.app.add_secret(juju_secret_content, label="vault-ca-certificate")
         peer_relation = self.model.get_relation(PEER_RELATION_NAME)
-        peer_relation.data[self.app].update({"vault-certificates-secret-id": juju_secret.id})  # type: ignore[union-attr]  # noqa: E501
-        logger.info("Vault certificate secret set in peer relation")
+        peer_relation.data[self.app].update({"vault-ca-certificates-secret-id": juju_secret.id})  # type: ignore[union-attr]  # noqa: E501
```

Comment:
Suggest to define labels and secret-id names ("vault-ca-certificates-secret-id", "vault-ca-certificate" ..etc) as global variables

Reply:
Done